package/xochitl/xochitl

```diff
@@ -2,7 +2,11 @@
 # Copyright (c) 2021 The Toltec Contributors
 # SPDX-License-Identifier: MIT
 
-echo "Loading custom environment variables from /opt/etc/xochitl.env.d"
-# shellcheck disable=SC2046
-export $(grep -vh '^#' /opt/etc/xochitl.env.d/* | /opt/bin/xargs -d '\n')
+for file in /opt/etc/xochitl.env.d/*; do
+    if [[ $file != /opt/etc/xochitl.env.d/README ]]; then
+        echo "Sourcing $file"
+        # shellcheck disable=SC1090
+        source "$file"
+    fi
+done
```

Comment:
I think this is probably a better solution overall. It would require the files to use the `.env` extension.
```suggestion
for file in /opt/etc/xochitl.env.d/*.env; do
    echo "Sourcing $file"
    # shellcheck disable=SC1090
    source "$file"
done
```